twig/twig/CVE-2022-23614.yaml

```diff
@@ -0,0 +1,11 @@
+title:     Disallow non closures in the sort filter
+link:      https://symfony.com/blog/twig-security-release-disallow-non-closures-in-the-sort-filter
+cve:       CVE-2022-23614
+branches:
+    2.x:
+        time:     2022-03-17 13:22:01
```

Comment:
This should be the time of the fix, https://github.com/advisories/GHSA-5mv2-rx3q-4w2v links to fix commits from Feb 4?

Reply:
@naderman fixed!